misc/test_site/mac_addrs/9a02571e8f01/device_report.md

```diff
@@ -0,0 +1,80 @@
+# DAQ scan report for device 9a02571e8f01
```

Comment:
extraneous file

Reply:
this file is getting auto-generated whenever I run 'cmd/run -s' or 'testing/test_aux.sh'
If this file shouldn't be in here, should it be added to the .gitignore? Or whatever process is creating this file, needs to be amended so that the file doesn't get created?

Reply:
I think you need this line (in your local settings):

site_reports=local/tmp

This is basically saying where to put generated site reports, so that it doesn't pollute the source repo.  It's already in the testing/test_aux.sh script... are you 100% sure they're being generated with that and not just some carry-over from a previous run?

Reply:
when i insert this line in my local/system.conf and 'run/cmd -s' these files doesn't get created. But when I run 'testing/test_aux.sh' - even though this line is present in the script - these files show up.